server/logical.sql

```diff
@@ -0,0 +1,68 @@
+DROP TABLE IF EXISTS users CASCADE; 
+DROP TABLE IF EXISTS cohort CASCADE; 
+DROP TABLE IF EXISTS events CASCADE; 
+DROP TABLE IF EXISTS recipes CASCADE; 
+DROP TABLE IF EXISTS dietary_restrictions CASCADE; 
+DROP TABLE IF EXISTS allergies CASCADE; 
+DROP TABLE IF EXISTS recipe_allergies CASCADE; 
+
+CREATE TABLE users(
+    id SERIAL PRIMARY KEY,
+    user_name VARCHAR(60) NOT NULL,
+    user_email VARCHAR(90) NOT NULL,
+    is_admin BOOLEAN NOT NULL,
+    is_lunch_maker BOOLEAN NOT NULL,
+    is_lunch_shopper BOOLEAN NOT NULL,
+    user_location VARCHAR(120) NOT NULL,
+    cohort_id INT REFERENCES cohort(id),
+    user_password VARCHAR(30) NOT NULL
+);
+
+CREATE TABLE cohort(
+    id SERIAL PRIMARY KEY,
+    class_number INT NOT NULL,
+    region VARCHAR(30) NOT NULL
+);
+
+CREATE TABLE events(
+    id SERIAL PRIMARY KEY,
+    meeting_location VARCHAR(60),
+    meeting_start VARCHAR(60),
+    meeting_end VARCHAR(60),
+    break_time VARCHAR(60),
+    lunch_maker_id INT REFERENCES users(id), 
+    lunch_shopper_id INT REFERENCES users(id),
+    budget VARCHAR(30),
+    meals INT,
+    recipe_id INT REFERENCES recipes(id) 
+);
+
+CREATE TABLE recipes(
+    id SERIAL PRIMARY KEY,
+    recipe_name VARCHAR(90) NOT NULL,
+    cooking_instructions VARCHAR(600) NOT NULL,
+    ingredients VARCHAR(360) NOT NULL,
```

Comment:
Should ingredients be a list here? Might be better to use either an `ARRAY` field or use another table for ingredients.